dataactcore/migrations/env.py

```diff
@@ -1,9 +1,13 @@
 from __future__ import with_statement
 from alembic import context
+# Load all DB tables into metadata object
+from dataactcore.models import domainModels
```

Comment:
do we need these imports? I had this file open in PyCharm, which says that domainModels, errorModels, jobModels, stagingModels, userModel, and validationModels are unused.